main.go

```diff
@@ -232,12 +239,60 @@ func main1() int {
 	return 0
 }
 
+func checkGoVersion() bool {
+	out, err := exec.Command("go", "version").CombinedOutput()
+	if err != nil {
+		fmt.Printf(`Can't get go version: %v
+
```

Comment:
Replace the possible reasons with this:

This is likely due to go not being installed/ setup correctly.

How to install go: https://golang.org/doc/install